x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -37,10 +46,14 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
     showAlertDetails,
     useFetchAlertData,
   }) => {
-    const { userCanCrud } = useCasesContext();
+    const { userCanCrud, features } = useCasesContext();
     const { metricsFeatures } = useCasesFeatures();
     useCasesTitleBreadcrumbs(caseData.title);
 
+    const { navigateToCaseView } = useCaseViewNavigation();
+    const { urlParams } = useUrlParams();
+    const activeTabId = urlParams.tabId || CASE_VIEW_PAGE_TABS.ACTIVITY;
```

Comment:
I found a small bug here. If you put a tabId in the URL that is different than the supported then you get the following view:

<img width="2309" alt="Screenshot 2022-04-29 at 1 24 58 PM" src="https://user-images.githubusercontent.com/7871006/165927740-245d5a7a-e93f-42f8-8afd-aeddb6e16256.png">
 
 I think we should defaulted to `activity` if the tabId is not supported.

Reply:
good one! I'll add a validation